### ISSUE TITLE
Maximum Likelihood Exercise 1

### ISSUE THREAD

**Issue by ToddMorrill** [d2l-ai/d2l-en#1772]
I think there might be a typo in [exercise 1 for the maximum likelihood chapter](https://d2l.ai/chapter_appendix-mathematics-for-deep-learning/maximum-likelihood.html#exercises). If you work out the calculations as it is, you'll get -1/3 for alpha, which results in negative pdf values. If the initial distribution is formulated as alpha*e^(-alpha*x) (aka the exponential distribution), everything works out.

**Comment by tuelwer** [comment]
I agree! I it should either be `\alpha e^{-\alpha x}` or `\frac{1}{\alpha} e^{-\frac{x}{\alpha}}` [(see here)](https://en.wikipedia.org/wiki/Exponential_distribution#Definitions). It might also be good to mention that the random variable only takes positive values (otherwise the density does not integrate to one which might be confusing for the reader.)
